pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -131,6 +131,43 @@ const (
 	retrySleepTime = 20 * time.Millisecond
 )
 
+const (
+	// These labels MUST match what defined in pkg/kubelet/apis/well_known_labels.go
+	labelOSBeta   = "beta.kubernetes.io/os"
```

Comment:
prefer to import "pkg/kubelet/apis" here.

Reply:
Done